docs/source/guide/tutorials/advanced/semi_sl.rst

```diff
@@ -2,6 +2,141 @@
+
+  (otx) ...$ otx train
+
+In the train log, you can check that the train type is set to **SEMISUPERVISED** and related configurations are properly loaded as following:
+
+.. code-block::
+
+  ...
+  2023-02-22 06:21:54,492 | INFO : called _init_recipe()
+  2023-02-22 06:21:54,492 | INFO : train type = SEMISUPERVISED
+  2023-02-22 06:21:54,492 | INFO : train type = SEMISUPERVISED - loading training_extensions/otx/recipes/stages/classification/semisl.yaml
+  2023-02-22 06:21:54,500 | INFO : Replacing runner from EpochRunnerWithCancel to EpochRunnerWithCancel.
+  2023-02-22 06:21:54,503 | INFO : initialized recipe = training_extensions/otx/recipes/stages/classification/semisl.yaml
+  ...
+
+
+After training ends, a trained model is saved in ``models`` sub-directory in workspace ``otx-workspace-CLASSIFICATION``.
+
+
+3. In the same manner with :doc:`the normal validation <../base/how_to_train/classification#validation>`, 
+we can evaluate the trained model in ``models`` folder with auto-splitted validation dataset 
+and save results to ``performance.json`` by the following command:
+
+.. code-block::
+
+  (otx) ...$ otx eval --test-data-roots splitted_dataset/val \
+                      --load-weights models/weights.pth \
+                      --save-performance performance.json
+
+***************************
+Enable via ``otx train``
+***************************
+
+1. To enable semi-supervised learning directly via ``otx train``, we need to add arguments ``--unlabeled-data-roots`` and ``--algo_backend.train_type`` 
+which is one of template-specific parameters (The details are provided in :doc:`quick start guide <../../get_started/quick_start_guide/installation>`.)
+
+.. code-block::
+
+  (otx) ...$ otx train otx/algorithms/classification/configs/mobilenet_v3_large_1_cls_incr/template.yaml \
+                      --train-data-roots data/flower_photos \
+                      --unlabeled-data-roots tests/assets/imagenet_dataset \
+                      params --algo_backend.train_type SEMISUPERVISED
+
+In the train log, you can check that the train type is set to **SEMISUPERVISED** and related configurations are properly loaded as following:
+
+.. code-block::
+
+  ...
+  2023-02-22 06:21:54,492 | INFO : called _init_recipe()
+  2023-02-22 06:21:54,492 | INFO : train type = SEMISUPERVISED
+  2023-02-22 06:21:54,492 | INFO : train type = SEMISUPERVISED - loading training_extensions/otx/recipes/stages/classification/semisl.yaml
+  2023-02-22 06:21:54,500 | INFO : Replacing runner from EpochRunnerWithCancel to EpochRunnerWithCancel.
+  2023-02-22 06:21:54,503 | INFO : initialized recipe = training_extensions/otx/recipes/stages/classification/semisl.yaml
+  ...
+
+
+After training ends, a trained model is saved in ``otx-workspace-CLASSIFICATION/models`` since OTX generates a workspace named ``otx-workspace-CLASSIFICATION`` in default.
+
+2. We can evaluate the trained model with auto-splitted validation dataset in the workspace and 
```

Comment:
It is more like stand-alone section. Could you add Validation section and output example for the given command below